Tribler/Core/torrentstore.py

```diff
@@ -130,5 +137,13 @@ def close(self):
         self._db = None
 
 
+    if LEVELDBPROVIDER == 'leveldb':
```

Comment:
@whirm 
you mean like this?
```python
_db = LevelDB(store_dir)    # class attr
...
def _get_write_batch(self):
    if _provider is 'leveldb':    # _provider is class attr, defined in try/except block
        return WriteBatch()
    elif _provider is 'plyvel':
        return WriteBatch(_db)
```

or do you think is there a better way of distinguishing which bindings are used?